src/commands/game_commands/preset.ts

```diff
@@ -72,10 +84,14 @@ export default class PresetCommand implements BaseCommand {
             return;
         }
 
-        const presetName = parsedMessage.components[1];
-        if (!presetName) {
+        const missingPresetMessage = (() => {
             sendErrorMessage(messageContext, { title: "Preset Name Missing", description: "You must specify a preset name.", thumbnailUrl: KmqImages.NOT_IMPRESSED });
             logger.warn(`${getDebugLogHeader(message)} | Preset name not specified`);
+        });
+
+        const presetName = parsedMessage.components[presetAction !== PresetAction.IMPORT ? 1 : 2];
+        if (presetAction !== PresetAction.IMPORT && !presetName) {
```

Comment:
Why do we still need this if it's already checked in the case statement

Reply:
If it were removed, in the case where neither the UUID nor the preset name are entered, it will only give the error message for the missing preset name, even though the UUID is the first argument